projects/plugins/boost/app/assets/src/js/pages/settings/sections/Score.svelte

```diff
@@ -112,6 +114,14 @@
 		$dismissedScorePromptStore = [ ...$dismissedScorePromptStore, id ];
 		dismissModal();
 	}
+
+	const panelStore = performanceHistoryPanelDS.store;
+	const onTogglePerformanceHistory = status => {
+		panelStore.set( status );
+	};
```

Comment:
`$panelStore = status`

Reply:
> I'm not sure if this is a rule - but we typically have named functions use the word function instead of being a const arrow function.

Gotcha. I've updated the syntax in ddcfc74.

> Although - if that function is called directly from an even, I think the status is probably actually in e.detail.status ? (e.detail is how svelte passes event values)

Thanks, but in this case it's react.